test_stack.py

```diff
@@ -0,0 +1,57 @@
+# Test file for stack.py
+# Authors Mark, Efrain, Henry
+
+import pytest
+import stack as st
+
+
+def test_init():
+    """Test stack constructor."""
+    a = st.Stack()
+    assert isinstance(a, st.Stack)
```

Comment:
this test is meaningless.  It would be impossible for Python to function if it were possible for this test to fail.